src/openms/source/FORMAT/PepXMLFile.cpp

```diff
@@ -777,12 +791,23 @@ namespace OpenMS
       //   current_peptide_.setScoreType(name); // add "X!Tandem" to name?
       //   current_peptide_.setHigherScoreBetter(true);
       // }
-      else if (name == "xcorr" && search_engine_ != "MyriMatch") // Sequest score; MyriMatch has also an xcorr, but we want to ignore it
+      else if (name == "xcorr") // Sequest score
       { // and use the mvh
         value = attributeAsDouble_(attributes, "value");
-        peptide_hit_.setScore(value);
-        current_peptide_.setScoreType(name); // add "Sequest" to name?
-        current_peptide_.setHigherScoreBetter(true);
+        if (search_engine_ != "MyriMatch") //MyriMatch has also an xcorr, but we want to ignore it
+        {
+          peptide_hit_.setScore(value);
+          current_peptide_.setScoreType(name); // add "Sequest" to name?
```

Comment:
I would not use Sequest as this does not distinguish between the different Sequest subscores.